ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -293,6 +293,7 @@ export const ApplicationsList = (props: RouteComponentProps<{}>) => {
     const [isAppCreatePending, setAppCreatePending] = React.useState(false);
     const loaderRef = React.useRef<DataLoader>();
     const {List, Summary, Tiles} = AppsListViewKey;
+    const [isStatusBarShown, showStatusBar] = React.useState(true);
```

Comment:
Actually, now that I'm thinking about this, if you make this change to your PR: 

```suggestion
    const [isStatusBarShown, showStatusBar] = React.useState(healthBarPrefs.showHealthStatusBar);
```

this may solve the problem and still have the local storage benefit

Reply:
You'll have to move the declaration of `healthBarPrefs` on line 363 to above

```ts
const healthBarPrefs = pref.statusBarView || ({} as HealthStatusBarPreferences);
```